docs/bugs.json

```diff
@@ -1,4 +1,11 @@
 [
+	{
+		"name": "KeccakCaching",
+		"summary": "The bytecode optimizer incorrectly evaluated certain keccak256 hashes. You are unlikely to be affected if you do not compute keccak256 hashes in inline assembly.",
```

Comment:
```suggestion
		"summary": "The bytecode optimizer incorrectly re-used previously evaluated keccak256 hashes. You are unlikely to be affected if you do not compute keccak256 hashes in inline assembly.",
```

Reply:
Nit: I think the Keccak-256 styling is used in writing in most places.